rest-api-spec/src/main/resources/rest-api-spec/test/index/100_partial_flat_object.yml

```diff
@@ -88,7 +88,16 @@ setup:
             } ]
```

Comment:
We have to change `skip` to `2.16.99` here, right?

Reply:
I was going to make that change on main let auto-backport do its thing.

Reply:
I'll open that PR right now

Reply:
There we go: https://github.com/opensearch-project/OpenSearch/pull/15448

("Right now" became an hour as I got distracted.)